data/kim2020viral/kim2020viral.yaml

```diff
@@ -0,0 +1,185 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Viral Load Kinetics of SARS-CoV-2 Infection in First Two Patients in Korea
+doi: 10.3346/jkms.2020.35.e86
+description: >
+  The authors present viral load kinetics for the first two confirmed COVID-19 patients
+  with mild to moderate illness in Korea. Swabs, sputum, serum, plasma, urine, and
+  stool samples were collected throughout the illness. Cycle threshold (Ct) values
+  were quantified using rRT-PCR targeting the RdRp and E genes. Data were obtained
+  from the supplementary material.
+analytes:
+  sputum_SARSCoV2_RdRp:
+    description: >
+      Cycle threshold (Ct) values were quantified using rRT-PCR targeting the RdRp
+      gene in sputum samples. Ct values were then converted into estimated viral copies
+      per mL, with a detection limit of 2,690 copies/mL. The RNA copy number was calculated
+      by the authors using a standard curve derived from Ct values of plasmid DNA,
+      as noted in the supplementary material.
+    limit_of_quantification: unknown
+    limit_of_detection: 2690
+    specimen: sputum
+    biomarker: SARS-CoV-2
+    gene_target: RdRp
+    unit: gc/mL
+    reference_event: symptom onset
+  swab_SARSCoV2_RdRp:
+    description: >
+      Cycle threshold (Ct) values were quantified using rRT-PCR targeting the RdRp
+      gene in swab samples. Ct values were then converted into estimated viral copies
+      per mL, with a detection limit of 2,690 copies/mL. The RNA copy number was calculated
+      by the authors using a standard curve derived from Ct values of plasmid DNA,
+      as noted in the supplementary material.
+    limit_of_quantification: unknown
+    limit_of_detection: 2690
+    specimen: throat_swab
```

Comment:
This is not `throat_swab` but a combined sample of Nasopharyngeal and oropharyngeal swabs. In the paper, it says: "Nasopharyngeal and oropharyngeal swabs were collected and placed in the same tube as URT specimen, and sputum was used as LRT specimen."

In the schema, we may add `nasopharyngeal_swab` and `nasal_swab` in the `specimen`.

Reply:
You're right! I plan to bring this up for discussion. For now, I temporarily changed the specimen to throat_swab to pass the validation check. I'm actually not clear on the distinction between nasopharyngeal_swab and nasal_swab, or the difference between oropharyngeal_swab and throat_swab. Since this is a combined sample, how should we handle it?

Reply:
For the combined sample, we could do something like this maybe?

```yaml
specimen_type:
  - oropharyngeal_swab
  - nasopharyngeal_swab
```

Reply:
Sounds good!